contrib/ci/void.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+set -x
+
+#install dependencies
+xbps-install -Suy python3
+./contrib/ci/generate_dependencies.py | xargs xbps-install -Suy
```

Comment:
-S and -u are unnecessary here, the repositories were synced and the system was updated before in the Dockerfile

Reply:
The Dockerfile is only updated nightly.  Is it possible they're out of sync?

Reply:
Although I guess the previous line installing python also did the sync and update.  So probably at most need it in dockerfile and when python installs and that's it.

Reply:
You are probably right, removing it from the second line shouldn't break anything. Anyway, xbps-install -Suy python3 will update **only** python3, nothing else.

Reply:
Fixed 

My experience was python3 wasn't in the container by default. I think after this is merged and the first nightly container builds it won't get needed anymore.

Reply:
What I am saying doesn't mean that you don't need to install python3, but that xbps-install -Suy python3 will not update the whole system. So add a line before this one with xbps-install -Suy.

I still don't think it's necessary to update the system again here. The Dockerfile is ran once per build right? If yes, then it doesn't matter that you are pulling a 24 hour old image, because in the next step you `RUN xbps-install -Suy`, so the system is up to date and -Su is not needed in this file.

Reply:
No the Dockerfile is used to generate the container once per night, and so `xbps-install -Suy` runs once per 24 hours.  From the CI this is the only command run:
https://github.com/fwupd/fwupd/blob/master/.github/workflows/main.yml#L55

Which effectively means it runs `void.sh` and that's it.

Reply:
That makes sense, thanks for explaining. You still need to run `xbps-install -Suy` somewhere though.

Reply:
I'll merge the PR as it is and then tomorrow after we have a successful run making the container will tear out python3 (which it shouldn't be needed anymore - should end up in container) and then can move the `-Suy` around